### ISSUE TITLE
Reproduction on the 3DPW dataset

### ISSUE THREAD

**Issue by mistrorigomanuel** [young0304/Adaptive-Spatial-Temporal-Graph-Mixer#1]
Dear authors,

I have been attempting to replicate the results reported in your paper using the provided codebase with the dataset 3DPW, but I am encountering difficulties in achieving the same performance levels. In particular, in your paper the mpjpe error is 25.3, 56.8, 79.9, 94.0, and 104.4 for 200, 400, 600, 800, and 1000 ms, respectively. Instead, after training and evaluating the model, I obtained 29.43, 60.55, 84.60, 101.44 and 113.35.
Im aware that running multiple times the same training script may provide different results, but the ones that I obtained are significantly different from the paper.

I have tried to specify all the model parameters as in the paper: 

parser.add_argument('--input_n', type=int, default=10, help="number of model's input frames")
parser.add_argument('--output_n', type=int, default=25, help="number of model's output frames")
parser.add_argument('--skip_rate', type=int, default=5, choices=[1, 5], help='rate of frames to skip,defaults=1 for H36M or 5 for AMASS/3DPW')
parser.add_argument('--num_worker', default=4, type=int, help='number of workers in the dataloader')
parser.add_argument('--r_se', default=8, type=int, required=False)
parser.add_argument('--n_epochs', default=50, type=int, required=False)
parser.add_argument('--batch_size', default=50, type=int, required=False)
parser.add_argument('--loader_workers', default=4, type=int, required=False)
parser.add_argument('--milestones', type=list, default=[10, 20, 30, 40], help='the epochs after which the learning rate is adjusted by gamma')
parser.add_argument('--gamma', type=float, default=0.1, help='gamma correction to the learning rate, after reaching the milestone epochs')
parser.add_argument('--batch_size_test', type=int, default=256, help='batch size for the test set')

parser_mpjpe.add_argument('--hidden_dim', default=50, type=int, required=False)
parser_mpjpe.add_argument('--num_blocks', default=3, type=int, required=False)
parser_mpjpe.add_argument('--tokens_mlp_dim', default=20, type=int, required=False)
parser_mpjpe.add_argument('--channels_mlp_dim', default=50, type=int, required=False)
parser_mpjpe.add_argument('--regularization', default=0.1, type=float, required=False)
parser_mpjpe.add_argument('--pose_dim', default=69, type=int, required=False)
parser_mpjpe.add_argument('--lr', default=0.01, type=float, required=False)

Another difference I have notices is the number of model parameters. I have 83909 params, while in the paper it is written 110k.

Could you please provide the detailed settings for training your model or help me to understand what is wrong?

Thanks,
Manuel
